docs/technical-overview.md

```diff
@@ -0,0 +1,267 @@
+*[➡️ Documentation index](./index.md)*
+
+# Technical Overview
+
+**Table of contents**
+
+* [Fixtures](#fixtures)
+* [Manufacturers](#manufacturers)
+* [Fixture model](#fixture-model)
+* [Fixture features](#fixture-features)
+* [Plugins](#plugins)
+  * [Exporting](#exporting)
+  * [Importing](#importing)
+  * [Export Tests](#export-tests)
+* [Testing](#testing)
+* [UI / Website](#ui--website)
+
+## Fixtures
+*[⬆️ Back to top](#technical-overview)*
+
+A fixture (sometimes also *personality*) is a lighting device that can be controlled with DMX. OFL gathers fixture definitions, that are specifications of a fixture's general information (like physical data) and the details of its [DMX](https://www.learnstagelighting.com/what-is-dmx-512/) controlment.
```

Comment:
A fixture **has** personalities (modes). So the fixture definitions are also called personalities, not the fixtures itself.